packages/eas-cli/src/build/create.ts

```diff
@@ -0,0 +1,147 @@
+import { EasJsonReader } from '@eas/config';
```

Comment:
The filename is confusing to me.

Reply:
it's an implementation of `build:create` command